packages/shared/src/libs/helpers/format-date/format-date.helper.ts

```diff
@@ -0,0 +1,16 @@
+import { differenceInHours, format } from "date-fns";
+
+import { DateValue } from "../../enums/enums.js";
+
+const formatDate = (inputDateString: string): string => {
```

Comment:
```suggestion
const formatDate = (inputDateString: string, type: valuesof<FormatDateType> ): string => {
```
add an enum for it and use it to pass to `format` function

Reply:
I am not even sure that you will need a `DateValue` then